### PR TITLE
docs: update mgt-todo documentation

### PR DESCRIPTION
**Pull request by Mnickii**
Closes https://github.com/microsoftgraph/microsoft-graph-toolkit/issues/2096

### CODE REVIEW

File: concepts/toolkit/components/todo.md [hunk 1]
```diff
@@ -137,8 +107,24 @@ This control uses the following Microsoft Graph APIs and permissions.
...
 
 ## Cache
 
-The `mgt-todo` component doesn't cache any data.
\ No newline at end of file
+The `mgt-todo` component doesn't cache any data.
+
+## Localization
+
+The control exposes the following variables that can be localized. See how you setup localization [here](../customize-components/localization.md).
```
Comment (Danielabom) [inline concepts/toolkit/components/todo.md:118]:
```suggestion
The control exposes the following variables that can be localized. For more information, see [setup localization](../customize-components/localization.md).
```

File: concepts/toolkit/components/todo.md [hunk 2]
```diff
@@ -137,8 +107,24 @@ This control uses the following Microsoft Graph APIs and permissions.
...
 ## Cache
 
-The `mgt-todo` component doesn't cache any data.
\ No newline at end of file
+The `mgt-todo` component doesn't cache any data.
+
+## Localization
+
+The control exposes the following variables that can be localized. For more information, see [setup localization](../customize-components/localization.md).
+
```
Comment (Danielabom) [inline concepts/toolkit/components/todo.md:119]: @musale Is this the valid format for default values? Why do you want to keep them in quotes?

**Review by Danielabom — CHANGES_REQUESTED** [review]
@musale, please review my comments. Thanks!

File: concepts/toolkit/components/todo.md (hunk 2 above)
Comment (Mnickii) [reply concepts/toolkit/components/todo.md:119]: Yes, this is the valid format for the default values, should be a back-tick though.

**Review by Danielabom — APPROVED** [review]
PR approved. Thanks

**Comment by Danielabom** [comment]
@Mnickii, please add the "ready to merge" label as soon as you want to publish the content.

**Pull request merged**